src/state/slices/marketDataSlice/marketDataSlice.tsx

```diff
@@ -166,9 +173,10 @@ export const marketApi = createApi({
         }
       },
     }),
-    findPriceHistoryByAssetId: build.query<HistoryData[], FindPriceHistoryByAssetIdArgs>({
+    findPriceHistoryByAssetId: build.query<HistoryData[] | null, FindPriceHistoryByAssetIdArgs>({
       queryFn: async (args, { dispatch }) => {
         const { assetId, timeframe } = args
+        if (ignoreAssetIds.includes(assetId)) return { data: null }
```

Comment:
```suggestion
        if (ignoreAssetIds.includes(assetId)) return { data: [] }
```

then we don't have to change the type signature